h2o-r/tests/testdir_algos/stackedensemble/runit_stackedensemble_custom_metalearner_params.R

```diff
@@ -0,0 +1,119 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+stackedensemble.custom.metalearner.test <- function() {
+
+    # This test checks the following (for binomial classification):
+    #
+    # 1) That h2o.stackedEnsemble `metalearner_algorithm` and `metalearner_params` work correctly
+    # 2) That h2o.stackedEnsemble `metalearner_algorithm` works in concert with `metalearner_nfolds`
+
+
+    train <- h2o.uploadFile(locate("smalldata/testng/higgs_train_5k.csv"),
+    destination_frame = "higgs_train_5k")
+    test <- h2o.uploadFile(locate("smalldata/testng/higgs_test_5k.csv"),
+    destination_frame = "higgs_test_5k")
+    y <- "response"
+    x <- setdiff(names(train), y)
+    train[,y] <- as.factor(train[,y])
+    test[,y] <- as.factor(test[,y])
+    nfolds <- 3  #number of folds for base learners
+
+    #Metalearner params for gbm, drf, glm, and deep deeplearning
+    gbm_params <- list(ntrees=100, max_depth = 6)
+    drf_params <- list(ntrees=100, max_depth = 6)
+    glm_params <- list(alpha=0, lambda=0)
+    dl_params  <- list(hidden=c(32,32,32), epochs = 20) ## small network, runs faster
+
+
+    # Train & Cross-validate a GBM
+    my_gbm <- h2o.gbm(x = x,
```

Comment:
nitpick: spacing, indent `y` and rest of params to where `x` is here and throughout the tests.

Reply:
This is done

Reply:
👍